packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -352,6 +355,43 @@ class TransactionService {
     )
   }
 
+  setSession(tr) {
+    let session = this._config.get('session')
+    if (session) {
+      if (typeof session == 'boolean') {
```

Comment:
Better to keep the object shape consistent here, first its boolean and then its object.

Reply:
The alternative here is to add another configuration option, I'm not against that idea either, but it seems unnecessary to me.